lib/app/view/app.dart

```diff
@@ -107,7 +107,7 @@ class _AppViewState extends State<AppView> {
           navigatorKey: _navigatorKey,
           debugShowCheckedModeBanner: false,
           title: "My Dairy",
-          supportedLocales: const [Locale('en')],
+          supportedLocales: const [Locale('en')], const [Locale('hi')],
```

Comment:
there is a syntax error in this line, change it to 
```
          supportedLocales: const [Locale('en'), Locale('hi')],
```

Reply:
done!